### PR TITLE
Task/01 setup

### PR DESCRIPTION
**Pull request by Vitalikkov**

### CODE REVIEW

**Review by MZabolotnev — CHANGES_REQUESTED** [review]
Check your deployment, page is empty

**Comment by Vitalikkov** [comment]
I fixed the problem, please try again @MZabolotnev

File: src/components/about/About.component.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import imageAbout from '@/assets/imageAbout.png';
+
+import styles from './about.module.css';
+
+const AboutComponent = () => (
+    <div className={styles.about_container}>
+        <h1 className={styles.about_title}>About me</h1>
+        <div className={styles.about_description}>
```
Comment (MZabolotnev) [inline src/components/about/About.component.tsx]: It can be more semantic. Pls use section / article instead of div

**Review by MZabolotnev — CHANGES_REQUESTED** [review]

**Comment by Vitalikkov** [comment]
I fixed it, please look again @MZabolotnev

File: src/components/about/About.component.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <p>This page was developed during the course &#39;Intro to React&#39; from Masters Academy in 2024.</p>
+            <p>
+                This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own
+                small project for the portfolio.
+            </p>
+            <p>You can contact me via [social network name] and check out my GitHub.</p>
+        </article>
+        <div className={styles.about_image}>
+            <img src={imageAbout} alt="me" />
+        </div>
```
Comment (MZabolotnev) [inline src/components/about/About.component.tsx]: So here also div. You can use figure instead

**Review by MZabolotnev — CHANGES_REQUESTED** [review]

File: src/components/about/About.component.tsx (hunk 2 above)
Comment (Vitalikkov) [reply src/components/about/About.component.tsx]: ✅

**Review by MZabolotnev — APPROVED** [review]

**Pull request merged**